res/layout/game.xml

```diff
@@ -27,11 +27,6 @@
     			android:layout_width="wrap_content"
 	    		android:layout_height="wrap_content"
 	    		android:text="New Game"/>
-    		<Button
```

Comment:
GitHub won't let me do file-level comments.

Canonically the layout files for activities are called activity_my_activity.xml.